vendor/k8s.io/kubernetes/plugin/cmd/kube-scheduler/app/server.go

```diff
@@ -124,7 +124,7 @@ func Run(s *options.SchedulerServer) error {
 	rl, err := resourcelock.New(s.LeaderElection.ResourceLock,
 		s.LockObjectNamespace,
 		s.LockObjectName,
-		leaderElectionClient,
+		leaderElectionClient.Core(),
```

Comment:
is this something we'll need to carry after a rebase onto 1.8? if so, indicate with `<carry>`

Reply:
No, 1.8 already has it.